WordPress/Classes/ViewRelated/Plans/PlanComparisonViewController.swift

```diff
@@ -0,0 +1,177 @@
+        let plan = viewController.plan
+        return allPlans.indexOf(plan)!
+    }
+    
+    private func updatePageControl() {
+        if let index = currentIndex {
+            pageControl?.currentPage = index
+        }
+    }
+}
+
+extension PlanComparisonViewController: UIPageViewControllerDataSource, UIPageViewControllerDelegate {
+    func pageViewController(pageViewController: UIPageViewController, viewControllerBeforeViewController viewController: UIViewController) -> UIViewController? {
+        return previousViewController()
+    }
+    
+    func pageViewController(pageViewController: UIPageViewController, viewControllerAfterViewController viewController: UIViewController) -> UIViewController? {
+        return nextViewController()
+    }
+    
+    private func previousViewController() -> UIViewController? {
+        guard let index = currentIndex else { return nil }
+        
+        return viewControllerAtIndex(index - 1)
+    }
+    
+    private func nextViewController() -> UIViewController? {
+        guard let index = currentIndex else { return nil }
+        
+        return viewControllerAtIndex(index + 1)
+    }
+    
+    func pageViewController(pageViewController: UIPageViewController, willTransitionToViewControllers pendingViewControllers: [UIViewController]) {
+        if let controller = pendingViewControllers.first as? PlanDetailViewController {
+            currentPlan = allPlans[indexOfViewController(controller)]
+        }
+    }
+    
+    func pageViewController(pageViewController: UIPageViewController, didFinishAnimating finished: Bool, previousViewControllers: [UIViewController], transitionCompleted completed: Bool) {
+        if let controller = pageViewController.viewControllers?.first as? PlanDetailViewController {
+            currentPlan = allPlans[indexOfViewController(controller)]
+            UIAccessibilityPostNotification(UIAccessibilityAnnouncementNotification, currentPlan.title)
+        }
+    }
+
+    /// `success` is if the scrolling action was successful (i.e. there was a page to scroll to)
+    private func scrollToNextPage(completion: ((success: Bool) -> ())? = nil) {
+        guard let nextViewController = nextViewController() else {
+            completion?(success: false)
+            return
+        }
+        
+        pageViewController.setViewControllers([nextViewController], direction: .Forward, animated: true, completion: nil)
+        completion?(success: true)
+    }
+    
+    /// `success` is true if the scrolling action was successful (i.e. there was a page to scroll to)
+    private func scrollToPreviousPage(completion: ((success: Bool) -> ())? = nil) {
+        guard let previousViewController = previousViewController() else {
+            completion?(success: false)
+            return
+        }
+        
+        pageViewController.setViewControllers([previousViewController], direction: .Reverse, animated: true, completion: nil)
+        completion?(success: false)
```

Comment:
I think this should be `true`. It's acting weird when you tap on the page control to go left.